### PR TITLE
Homepage video & Page element

### PR DESCRIPTION
**Pull request by NofarGizra**
#33

ibli

### CODE REVIEW

File: ibli/modules/custom/ibli_migrate/csv/node/.~lock.page.csv# [hunk 1]
```diff
@@ -0,0 +1 @@
+Nofar-PC ,nofar-pc,nofarpc-desktop,12.05.2014 09:47,file:///home/nofar-pc/.config/libreoffice/3;
```
Comment (bricel) [inline ibli/modules/custom/ibli_migrate/csv/node/.~lock.page.csv#:1]: Remove this file

File: ibli/modules/custom/ibli_migrate/handlers/node/IbliPageNodes.php [hunk 2]
```diff
@@ -3,12 +3,11 @@
 /**
```
Comment (bricel) [inline ibli/modules/custom/ibli_migrate/handlers/node/IbliPageNodes.php:3]: Filename should .inc not .php

**Comment by bricel** [comment]
Merged

File: ibli/modules/custom/ibli_migrate/handlers/node/IbliPageNodes.php (hunk 2 above)
Comment (amitaibu) [inline ibli/modules/custom/ibli_migrate/handlers/node/IbliPageNodes.php:3]: We are now following d8 convention, so php ending is fine

‫ב-13 במאי 2014, בשעה 16:44, ‏Brice Lenfant notifications@github.com כתב/ה:‬

**Pull request closed without merging**